packages/plugin-value-from-package/src/index.js

```diff
@@ -22,9 +22,14 @@ async function valueFromPackage({
   value: string,
   path: ?string,
 }): Promise<string> {
-  return JSON.parse(await fs.readFile(`${path || ''}package.json`, 'utf8'))[
+  const packagePath = `${path || ''}package.json`;
+  const packageValue = JSON.parse(await fs.readFile(packagePath, 'utf8'))[
     value
   ];
+  if (typeof packageValue === 'undefined') {
```

Comment:
This makes me very uncomfortable, but undefined is not valid JSON (let's hope that spec holds), so alright